corehq/apps/registration/views.py

```diff
@@ -199,11 +199,6 @@ def dispatch(self, request, *args, **kwargs):
             else:
                 return redirect("homepage")
         response = super(UserRegistrationView, self).dispatch(request, *args, **kwargs)
-        if settings.IS_SAAS_ENVIRONMENT:
```

Comment:
please be very careful that this workflow is not visible to our ENTERPRISE environments!